recipes/archicad-apidevkit/all/conanfile.py

```diff
@@ -0,0 +1,36 @@
+from conan import ConanFile
+from conan.tools.files import get
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.00"
+
+
+class ArchicadApidevkitConan (ConanFile):
+    name = "archicad-apidevkit"
+    description = "The General API Development Kit enables software developers to extend the functionality of Archicad"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://archicadapi.graphisoft.com/"
+    license = "https://archicadapi.graphisoft.com/documentation/license-agreement"
```

Comment:
```suggestion
    license = "https://archicadapi.graphisoft.com/documentation/license-agreement"
```

This requires 2 changes:
- Download a license copy, which should be packaged together. You can use [ApprovalTest](https://github.com/conan-io/conan-center-index/blob/master/recipes/approvaltests.cpp/all/conandata.yml#L5) as example
- The license attribute should follow SPDX format. As this license is not a regular open source, it should use `LicenseRef-<filename>`. More info: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-license-should-i-use-for-a-custom-project-specific-license